README.md

```diff
@@ -83,17 +85,40 @@ func criticalSection(ctx context.Context) {
 **HTTP Handler**
 
 ```go
-func main() {
-  http.Handle("/", xray.Handler(xray.NewFixedSegmentNamer("myApp"), http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+package main
```

Comment:
From my opinion, I don't think it's a good idea to add library information into every example. Lots of code are just repeated.

Reply:
The problem that I had (recently when I started to need to integrate x-ray again) is that I couldn't find these easy to do. Maybe a better way would be to have a few [testable examples](https://blog.golang.org/examples) instead? I'm happy to redo the PR if so.

Reply:
In README configuration section, there is a part to tell customer to import xray library. Have you take a look at that? Could you please tell me which example you were getting stuck into?